Dockerfile

```diff
@@ -10,11 +10,19 @@ RUN dnf update -y && \
         libguestfs-devel && \
     dnf clean -y all
 
+# Create tarball for the appliance. This fixed libguestfs appliance uses the root in qcow2 format as container runtime not always handle correctly sparse files. This appliance can be extracted and copied directly in the container image.
 RUN mkdir -p /output && \
-    cd /output && \
-    libguestfs-make-fixed-appliance --xz && \
-    KERNEL_VERSION=$(rpm -qa kernel-core | sed 's/kernel-core-\(.*\)\.el9.*/\1/') && \
+    mkdir -p /appliance && \
+    libguestfs-make-fixed-appliance /appliance && \
+    cd /appliance && \
+    mv root root.raw && \
+    qemu-img convert -c -O qcow2 root.raw root && \
+    rm root.raw
+
+COPY BUILD /appliance/BUILD
```

Comment:
How is the `BUILD` file used? Is it supposed to be picked up by KubeVirt's build system somehow?

Actually, it would be nice to see the corresponding changes in KubeVirt so that the entire thing can be reviewed holistically. Or maybe since you're no longer changing the compression algorithm things are supposed to be fairly transparent on the KubeVirt side?

Reply:
yes, I'm struggling getting the bazel part right... But then I plan to publish a WIP pr with the changes before merging this ;)

Reply:
@andreabolognani here: https://github.com/kubevirt/kubevirt/pull/8734